src/github/tests/test_utils.rs

```diff
@@ -188,6 +350,10 @@ pub struct GithubMock {
     team_memberships: HashMap<String, HashMap<UserId, TeamMember>>,
     // Team name -> list of invited users
     team_invitations: HashMap<String, Vec<String>>,
+    // Repo name -> repo data
+    repos: HashMap<String, Repo>,
+    // Repo name -> (teams, members)
+    repo_members: HashMap<String, (Vec<RepoTeam>, Vec<RepoUser>)>,
```

Comment:
to improve clarity, we could turn `(Vec<RepoTeam>, Vec<RepoUser>)` into a struct:

```rs
struct RepoMembers {
    teams: Vec<RepoTeam>,
    users: Vec<RepoUser>,
}
```

wdyt?

Reply:
Sure, why not :)